components/gitpod-cli/cmd/top.go

```diff
@@ -22,7 +23,12 @@ import (
 	"github.com/olekukonko/tablewriter"
 )
 
+var opts struct {
+	Json bool
+}
+
 func outputTable(workspaceResources *supervisor.ResourcesStatusResponse) {
+	fmt.Printf("Workspace Class: %s\n", workspaceResources.WorkspaceClass)
```

Comment:
It would be nice also to print its configuration in human readable form, i.e. minimal and max CPU. To avoid confusion why CPU limit is shifting. Not sure how to present it, some suggested to put it as limit instead of real cgroup limit, but it can be confusing if a user did not manage to get it.

I remember @Furisto mentioned that we can pass it to supervisor from worksapcekit via env var. Maybe can be another PR as well.